source/Ship.cpp

```diff
@@ -1775,7 +1775,7 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 	if(acceleration)
 	{
 		acceleration *= slowMultiplier;
-		Point dragAcceleration = acceleration - velocity * (attributes.Get("drag") / mass);
+		Point dragAcceleration = acceleration - velocity * (attributes.Get("drag") / (mass/60));
```

Comment:
😔 No one ever reads the [style guide](https://endless-sky.github.io/styleguide/styleguide.xml).
```suggestion
		Point dragAcceleration = acceleration - velocity * (attributes.Get("drag") / mass / 60.);
```

Reply:
So acceleration is m/s^2. Velocity is m/s. Drag is kg*m/s^2, as it is a force. Mass is kg. The added 60 is s. That means in terms of units we get:
* [m/s^2] - [s] * [m/s] * [kg*m/s^2] / [kg]
* = [m/s^2] - [s] * [m/s] * [m/s^2]
* = [m/s^2] - [s] * [m^2/s^3]
* = [m/s^2] - [m^2/s^2]
Cool.

Reply:
Sorry, first time code edit. My B.
I went with the division because then it lines up conceptually better with converting displayed acceleration into per-frame acceleration and it's specifically that element that's out of step - even though it doesn't make any mathematical difference.